README.md

```diff
@@ -78,96 +98,30 @@ After downloading the repository, do the following:
-- On OSX with multiple Python installs (e.g. via brew) cmake might erroneously pick various pieces from different installs which will likely produce abort trap at runtime. Remove cmake cache and re-run cmake with  `-DPYTHON_LIBRARY=/path/to/lib/libpython2.7.dylib` and  `-DPYTHON_INCLUDE_DIR=/path/to/include/python2.7` options to override with desired Python install path.
-- To use Include What You Use during compilation, pass `-DNUPIC-IWYU=ON`. This requires that IWYU is installed and findable by CMake, with a minimum CMake version of 3.3. IWYU can be installed from https://include-what-you-use.org/ for Windows and Linux, and on OS X using https://github.com/jasonmp85/homebrew-iwyu.
-- If you would like to install all headers, libraries, and executables to the install location for C++ clients when building the nupic.binding python extensions, pass `-NUPIC_TOGGLE_INSTALL=ON`.
-
-
-#### Build:
-
-    # While still in $NUPIC_CORE/build/scripts
-    make -j3
-
-> **Note**: The `-j3` option specifies '3' as the maximum number of parallel jobs/threads that Make will use during the build in order to gain speed. However, you can increase this number depending your CPU.
-
-#### Install:
-
-    # While still in $NUPIC_CORE/build/scripts
-    make install
-
-#### Run the tests:
-
-    cd $NUPIC_CORE/build/release/bin
-    ./unit_tests
-    ...
-
-#### Install nupic.bindings Python library:
-
-    cd $NUPIC_CORE
-    python setup.py develop
-
-> **Note**: If the extensions haven't been built already then this will call the cmake/make process to generate them.
-
-If you get a gcc exit code 1, you may consider running this instead:
-
-     python setup.py develop --user
-
-If you are installing on Mac OS X, you must add the instruction `ARCHFLAGS="-arch x86_64"` before the python call:
-
-    ARCHFLAGS="-arch x86_64" python setup.py develop
-
-Alternatively, you can use the `install` command (as opposed to `develop`) to copy the installation files rather than link to the source location.
-
-    python setup.py install
-
-> _Note_: If you get a "permission denied" error when using the setup commands above, you may add the `--user` flag to install to a location in your home directory, which should resolve any permissions issues. Doing this, you may need to add this location to your PATH and PYTHONPATH.
-
-Once it is installed, you can import NuPIC bindings library to your python script using:
-
-    import nupic.bindings
-
-You can run the nupic.bindings tests via `setup.py`:
-
-    python setup.py test
+If you get no error then python is able to load the nupic extension libraries.
 
+There are two sets of unit tests.
+ * C++ Unit tests -- to run: `cd build/Release/bin; ./unit_tests`
+ * Python Unit tests -- to run: `python setup.py test`
 ### Using graphical interface
 
 #### Generate the IDE solution:
```

Comment:
this applies to Linux/OSX IDEs (Netbeans, XCode, Eclipse..I think same would be for MSVC) 
Basically it says: 
"Select project that configures using CMake, setup sources src/ and build build/srcipts/ folders, run CMake as .. " 
It allows you to open this repo as a project in almost any IDE